packages/dropdowns/src/styled/field/StyledMultiselect.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css } from 'styled-components';
+import { getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledInput } from './StyledInput';
+import { SelectWrapper } from './StyledSelect';
+
+/**
+ * TODO: themify these values
```

Comment:
Is themifying planned for the PR following this one?  I see that some CSS properties are using `theme`.